pygpudrive/env/env_torch.py

```diff
@@ -196,8 +213,9 @@ def get_obs(self):
         return obs_filtered
 
     def get_controlled_agents_mask(self):
-        """Get the control mask."""
-        return (self.sim.controlled_state_tensor().to_torch() == 1).squeeze(
+        """Get the control mask. Bicycle = 1, DeltaModel = 2"""
+        target_idx = 2 if self.action_type == 'delta' else 1
+        return (self.sim.controlled_state_tensor().to_torch() == target_idx).squeeze(
```

Comment:
I don't think the `self.sim.controlled_state_tensor().to_torch()` ever contains values of 2? How does this work?

Reply:
`controlled_state` is decided with `ControlMode`. There are three `ContorlMode: EXPERT, BICYCLE, DELTA`. You can see this code in `src/types.hpp`.